servers/physics_server_3d.h

```diff
@@ -799,7 +799,12 @@ class PhysicsServer3D : public Object {
 
 	/* MISC */
 
-	virtual void free(RID p_rid) = 0;
+	virtual void free_rid(RID p_rid) = 0;
+#ifndef DISABLE_DEPRECATED
+	[[deprecated]] virtual void free(RID p_rid) {
```

Comment:
Sorry, i only saw this on the second look.
The deprecated alias shouldn't be virtual. Otherwise, we risk someone overriding `free` without overriding `free_rid`, leading to problems.

Reply:
Ah, true.